concourse/pipelines/deploy-cloudfoundry.yml

```diff
@@ -234,118 +225,173 @@ jobs:
     serial: true
     plan:
       - aggregate:
+        - get: paas-cf
+          passed: ['generate-manifest']
+        - get: cf-manifest
+          passed: ['generate-manifest']
+          trigger: true
+        - get: bosh-secrets
+      - aggregate:
         - task: stemcell-tarball
           config:
             params:
-              STEMCELL_VERSION: {{stemcell-version}}
-            image: docker:///governmentpaas/curl-ssl
+              VERSION: {{stemcell-version}}
+              NAME: bosh-aws-xen-hvm-ubuntu-trusty-go_agent
+              URL: https://bosh.io/d/stemcells/"${NAME}"?v="${VERSION}"
+              BOSH_PASSWORD: {{bosh_password}}
+              TYPE: stemcell
+            image: docker:///concourse/bosh-deployment-resource
             platform: linux
+            inputs:
+            - name: paas-cf
+            - name: bosh-secrets
             run:
               path: sh
-              args: ["-c", "curl -L -J -o stemcell.tgz https://s3.amazonaws.com/bosh-jenkins-artifacts/bosh-stemcell/aws/light-bosh-stemcell-${STEMCELL_VERSION}-aws-xen-hvm-ubuntu-trusty-go_agent.tgz"]
-
+              args:
+              - -e
+              - -c
+              - |
+                ./paas-cf/concourse/scripts/bosh_ensure_uploaded.sh
         - task: cf-release-tarball
           config:
             params:
-              RELEASE_VERSION: {{cf-release-version}}
-              URL: https://bosh.io/d/github.com/cloudfoundry/cf-release?v=${RELEASE_VERSION}
+              VERSION: {{cf-release-version}}
+              NAME: cf
+              URL: https://bosh.io/d/github.com/cloudfoundry/cf-release?v="${VERSION}"
+              BOSH_PASSWORD: {{bosh_password}}
+              TYPE: release
+            image: docker:///concourse/bosh-deployment-resource
             platform: linux
-            image: docker:///governmentpaas/curl-ssl
+            inputs:
+            - name: paas-cf
+            - name: bosh-secrets
```

Comment:
Remove this, as mention in https://github.com/alphagov/paas-cf/pull/82/files#r51403781